drivers/ethernet/eth_stm32_hal.c

```diff
@@ -196,14 +230,28 @@ static int eth_tx(const struct device *dev, struct net_pkt *pkt)
 	}
 
 #if defined(CONFIG_SOC_SERIES_STM32H7X)
-	const uint32_t cur_tx_desc_idx = 0;  /* heth->TxDescList.CurTxDesc; */
+	const uint32_t cur_tx_desc_idx = heth->TxDescList.CurTxDesc;
```

Comment:
Nice. I opened an issue asking why they had this as 0, don't think I ever heard back.

Reply:
🐛